src/slack/constants.ts

```diff
@@ -0,0 +1 @@
+export const registerTeamCallbackId = 'registerTeam';
```

Comment:
@johnkahn I tried using a `d.ts` file here but didn't have any luck... I assume you can't `declare` constants like you can with interfaces?

Reply:
Yeah you can't, and I was wrong about the standard with types anyway. Because we are inside of a typescript project, we shouldn't use `.d.ts` files. Those are only if you code in Javascript. So for now always use just `.ts` for types/constants